### PR TITLE
refactoring of game buttons with more storybook examples

### PR DESCRIPTION
**Pull request by pcardune**
I'm working on adding a reset button to the gamelab embed view and was really confused about how rendering of these buttons worked. So I did a bit of refactoring and added some more storybook examples.

Some important things that happened:
* We no longer use inline styles. They actually were duplicates of what was already in common.scss and they greatly complicated the code with lots of `isMinecraft` flags.
* I added support for including scss files directly from js so that I could load a custom scss file that scopes the minecraft styles to just the table cells that I want.

See the following screenshot for an example of what the reset button looks like:

screen shot 2017-01-03 at 5 16 05 pm

### CODE REVIEW

File: apps/package.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
     "build:log": "grunt logBuildTimes",
     "build:stats": "node ./script/build-stats.js",
     "prestorybook": "curl -o build/package/css/application.css http://localhost-studio.code.org:3000/assets/application.css",
-    "storybook": "start-storybook -p 9001 -s build/package/",
+    "storybook": "start-storybook -p 9001 -s build/package/,../dashboard/public",
```
Comment (bcjordan) [inline apps/package.json:30]: Ah nice, does this mean storybook will refresh on (dashboard?) stylesheet changes?

File: apps/src/templates/GameButtons.jsx [hunk 2]
```diff
@@ -16,16 +15,6 @@ const styles = {
     // to gameButtons. We want to get rid of that
     marginBottom: -18
   },
-  runImage: {
```
Comment (bcjordan) [inline apps/src/templates/GameButtons.jsx:19]: Don't see re-addition elsewhere, assuming for non-minecraft apps this set of styles was never removed from the stylesheet?

Comment (pcardune) [reply apps/src/templates/GameButtons.jsx:19]: correct

File: apps/package.json (hunk 1 above)
Comment (pcardune) [reply apps/package.json:30]: not automatically. You'll need to hit refresh in your browser.

**Pull request merged**